stdlib/sys.pyi

```diff
@@ -292,6 +305,10 @@ if sys.version_info < (3, 9):
     def getcheckinterval() -> int: ...  # deprecated
     def setcheckinterval(__n: int) -> None: ...  # deprecated
 
+if sys.version_info < (3, 9):
+    # An 11-tuple or None
+    def callstats() -> tuple[int, int, int, int, int, int, int, int, int, int, int] | None: ...
```

Comment:
In 3.7 and 3.8 it actually always returned None. This was never documented as far as I can tell.

Reply:
If it always returned `None`, why the tuple return type?

Reply:
Because we still support Python 3.6.